huggingface/base/inference/docker/py3.7/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,107 @@
+FROM nvidia/cuda:11.0-cudnn8-runtime-ubuntu18.04
```

Comment:
Is SageMaker Hosting now supporting Cuda 11? I thought it was only Cuda 10.2?

Reply:
I am currently testing this. The CUDA interoperability is more nuanced than I thought. 😄 . Will update this if endpoint creation fails.